app/ports/management/commands/update-rsync.py

```diff
@@ -0,0 +1,46 @@
+import subprocess
+import json
+
+from django.core.management.base import BaseCommand, CommandError
+
+from parsing_scripts import git_update
+from ports.models import Port, Commit
+
+RSYNC = "rsync://rsync.macports.org/macports//trunk/dports/PortIndex_darwin_16_i386/PortIndex.json"
+JSON_FILE = "portindex.json"
+
+
+class Command(BaseCommand):
+    help = "Populates the database with Initial data from portindex.json file"
+
+    def add_arguments(self, parser):
+        parser.add_argument('new',
+                            nargs='?',
+                            default=False,
+                            help="Define a commit till which the update should be processed")
+        parser.add_argument('old',
+                            nargs='?',
+                            default=False,
+                            help="Not recommended. Helps you provide a commit from which update should start")
+
+    def handle(self, *args, **options):
+        # fetch from rsync
+        subprocess.call(['/usr/bin/rsync', RSYNC, JSON_FILE])
```

Comment:
The path to `rsync` should be defined in the above mentioned configuration.
The path to json file should be absolute rather than relative.
I would probably use `-ac` argument to rsync or something similar to sync the files.

Reply:
OKay, thanks